### PR TITLE
[browser] Update runtime's DevServer to match Blazor's updates in 9

### LINKED ISSUE #105025: [browser] Update runtime's DevServer to match Blazor's updates in 9

### PR DESCRIPTION
**Pull request by maraf**
Fixes https://github.com/dotnet/runtime/issues/105025

### CODE REVIEW

File: src/mono/wasm/host/DevServer/DevServerStartup.cs [hunk 1]
```diff
@@ -57,7 +57,9 @@ public static void Configure(IApplicationBuilder app, IOptions<DevServerOptions>
             });
         }
 
-        app.UseBlazorFrameworkFiles();
+        //app.UseBlazorFrameworkFiles();
```
Comment (ilonatommy) [inline src/mono/wasm/host/DevServer/DevServerStartup.cs:60]: leaving in a form of a comment?

File: src/mono/wasm/host/DevServer/ComponentsWebAssemblyApplicationBuilderExtensions.cs [hunk 2]
```diff
@@ -1,139 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
```
Comment (ilonatommy) [inline src/mono/wasm/host/DevServer/ComponentsWebAssemblyApplicationBuilderExtensions.cs:1]: Where were we using this file and why we don't need it anymore? I can't see any other usage in the repo

File: src/mono/wasm/host/BrowserHost.cs [hunk 3]
```diff
@@ -13,6 +13,7 @@
 using System.Threading.Tasks;
 using System.Web;
 using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc.ApplicationParts;
```
Comment (ilonatommy) [inline src/mono/wasm/host/BrowserHost.cs:16]: Which added line requires this using?

**Review by ilonatommy — APPROVED** [review]

File: src/mono/wasm/host/DevServer/DevServerStartup.cs (hunk 1 above)
Comment (maraf) [reply src/mono/wasm/host/DevServer/DevServerStartup.cs:60]: I'm copying changes from Blazor. They have it commented out. I left it the same for better code matching...

File: src/mono/wasm/host/BrowserHost.cs (hunk 3 above)
Comment (maraf) [reply src/mono/wasm/host/BrowserHost.cs:16]: None, thanks!

**Review by lewing — APPROVED** [review]

**Pull request merged**